client/src/addSighting.jsx

```diff
@@ -3,13 +3,13 @@ import ReactDOM from 'react-dom';
 import '../style.sass';
 
 export class NewSighting extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            animal: '',
-            location: '',
-            time: '',
-            image: ''
+	constructor(props) {
+		super(props);
+		this.state = {
```

Comment:
I don't think this indentation is correct?

Reply:
There were a few that were still spaces for some reason, I've switched them to tabs. Otherwise, this matches all the other files (this.state tabbed in once from constructor(props)